app/views/products/index.html.erb

```diff
@@ -32,4 +32,10 @@
 
 <br>
 
-<%= link_to 'Создать новый продукт', new_product_path %>
+<%= link_to 'Создать новый продукт', new_product_path %> 
+<p></p>
+<% if current_user.nil? %>
+  <%= link_to new_user_session_path, class: 'login-button' do %>Sign in<% end %>
+<% else %>
```

Comment:
This have to move to general layout, so its available all over the site

Reply:
Moved in general layout